pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTPipelineDef.java

```diff
@@ -22,6 +22,7 @@
     private ModelASTBuildParameters parameters;
     private ModelASTTriggers triggers;
     private ModelASTLibraries libraries;
+    private ModelASTAxes axes;
```

Comment:
Like I mentioned in my first comment, I think the way to go here is to actually build this on top of the sequential stages in #227, as a `stage`-level construct only.